netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -129,6 +134,16 @@ public NettyChannelBuilder flowControlWindow(int flowControlWindow) {
     return this;
   }
 
+  /**
+   * Sets the maximum message size allowed to be received on the channel. If not called,
+   * defaults to 100 MiB.
+   */
+  public NettyChannelBuilder maxMessageSize(int maxMessageSize) {
+    checkArgument(maxMessageSize > 0, "maxMessageSize must be positive");
```

Comment:
It is true. 0 is a valid value. It is about as useful as 1 byte is, but it is valid.